spec/inherited_class_var_spec.rb

```diff
@@ -34,32 +34,6 @@ class InheritedBaseClass; end
         expect(Parent.inherited_hash).to eql(test1: 'test1', test2: 'test2')
         expect(Parent.inherited_hash.object_id).to eql Parent.inherited_hash.object_id
       end
-
-      context 'with cache cleared' do
-        before do
-          # NOTE deep_clear_class_cache doesn't work here...
-          [ :@_inherited_hash, :@_inherited_hash_inherited_class_cache ].each do |variable|
-            Parent.instance_variable_set(variable, nil)
-          end
-          # Parent.send(:deep_clear_class_cache, :@_inherited_hash)
-        end
-
-        it 'inherited merge' do
-          Parent.merge_inherited_hash(test1: 'test1')
-          expect(Parent.inherited_hash).to eql(test1: 'test1')
-
-          ClassWithFamily.merge_inherited_hash(test2: 'test2')
-          expect(ClassWithFamily.inherited_hash).to eql(test1: 'test1', test2: 'test2')
-        end
-
-        it 'merge options', skip: true do
-          Parent.merge_inherited_hash(test1: { foo: 'test1' })
-          expect(Parent.inherited_hash[:test1]).to eql({ foo: 'test1' })
-
-          ClassWithFamily.merge_inherited_hash(test1: { bar: 'test2' })
-          expect(ClassWithFamily.inherited_hash[:test1]).to eql({ foo: 'test1', bar: 'test2' })
-        end
-      end
```

Comment:
i don't think this is needed?

i think you were confused about what the caching does. clearing the cache won't change the values of the `inherited_class_var`, it just forces a recalculation in the block: https://github.com/FinalCAD/inherited_class_var/blob/e6a20ce15933dbec12e1a3be8a2c2b6fbcf95b41/lib/inherited_class_var.rb#L72-L76

i wrote a comment to explain why i wrote the caching code here: https://github.com/FinalCAD/inherited_class_var/pull/3/commits/9909414127e1deee9505f4399a6dc9707b0056f4